integration/spark/src/test/scala/org/apache/carbondata/integration/spark/testsuite/emptyrow/TestSkipEmptyLines.scala

```diff
@@ -33,50 +33,65 @@ class TestSkipEmptyLines extends QueryTest with BeforeAndAfterAll {
-    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' into table skipEmptyRowCarbonTable " +
-        s"OPTIONS('skip_empty_line'='true')")
-    checkAnswer(sql("select * from skipEmptyRowCarbonTable"), Seq(Row("a",25),Row("b",22),Row("c",23)))
+    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' " +
+        "into table skipEmptyRowCarbonTable OPTIONS('skip_empty_line'='true')")
+    checkAnswer(sql("select * from skipEmptyRowCarbonTable"),
+      Seq(Row("a", 25), Row("b", 22), Row("c", 23)))
   }
 
   test("test load options with false") {
     sql("drop table if exists skipEmptyRowCarbonTable")
     sql("CREATE TABLE skipEmptyRowCarbonTable (name string, age int) STORED AS carbondata")
-    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' into table skipEmptyRowCarbonTable " +
-        s"OPTIONS('skip_empty_line'='false')")
+    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' " +
+        "into table skipEmptyRowCarbonTable OPTIONS('skip_empty_line'='false')")
     checkAnswer(sql("select * from skipEmptyRowCarbonTable"),
-      Seq(Row("a",25),Row("b",22),Row("c",23),Row(null,null),Row(null,null),Row(null,null)))
+      Seq(Row("a", 25),
+        Row("b", 22),
+        Row("c", 23),
+        Row(null, null),
+        Row(null, null),
+        Row(null, null)))
   }
 
   test("test carbonproperties with true") {
     CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_SKIP_EMPTY_LINE, "true")
     sql("drop table if exists skipEmptyRowCarbonTable")
     sql("CREATE TABLE skipEmptyRowCarbonTable (name string, age int) STORED AS carbondata")
-    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' into table skipEmptyRowCarbonTable")
+    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' " +
+        "into table skipEmptyRowCarbonTable")
     checkAnswer(sql("select * from skipEmptyRowCarbonTable"),
-      Seq(Row("a",25),Row("b",22),Row("c",23)))
+      Seq(Row("a", 25), Row("b", 22), Row("c", 23)))
     CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_SKIP_EMPTY_LINE,
       CarbonCommonConstants.CARBON_SKIP_EMPTY_LINE_DEFAULT)
   }
 
   test("test carbonproperties with false") {
-    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_SKIP_EMPTY_LINE, "false")
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.CARBON_SKIP_EMPTY_LINE, "false")
     sql("drop table if exists skipEmptyRowCarbonTable")
     sql("CREATE TABLE skipEmptyRowCarbonTable (name string, age int) STORED AS carbondata")
-    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' into table skipEmptyRowCarbonTable")
+    sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/emptylines.csv' " +
+        "into table skipEmptyRowCarbonTable")
     checkAnswer(sql("select * from skipEmptyRowCarbonTable"),
-      Seq(Row("a",25),Row("b",22),Row("c",23),Row(null,null),Row(null,null),Row(null,null)))
+      Seq(Row("a", 25),
```

Comment:
same as above comment. handle for line 106

Reply:
changed for all places